src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -449,7 +449,7 @@ public void Enqueue(object callback, bool forceGlobal)
         {
             Debug.Assert((callback is IThreadPoolWorkItem) ^ (callback is Task));
 
-            if (loggingEnabled)
+            if (FrameworkEventSource.Log.IsEnabled() && loggingEnabled)
```

Comment:
Same as my other comment:
```suggestion
            if (loggingEnabled && FrameworkEventSource.Log.IsEnabled())
```